src/common/models/Projection.ts

```diff
@@ -372,24 +382,23 @@ export class Projection {
   }
 
   /**
-   * G
+   * Gets the cases/day on day i, by averaging the trailing week of cases/day
+   * data.
```

Comment:
can we add a reason why we are doing this? here

Reply:
Oh, yeah.  I just added the comment because it was missing, but then I forgot to document my change. Hah.  Done.